homeassistant/components/cloud/account_link.py

```diff
@@ -125,7 +132,8 @@ async def await_tokens():
     async def async_resolve_external_data(self, external_data: Any) -> dict:
         """Resolve external data to tokens."""
         # We already passed in tokens
-        return external_data
+        dict_data: dict = external_data
+        return dict_data
```

Comment:
Why not change `async_resolve_external_data(self, external_data: Any)` to `async_resolve_external_data(self, external_data: dict)` instead?
Its directly returning it.

Reply:
That's not compatible with the superclass definition: 
https://github.com/home-assistant/core/blob/2ebec14fe4bcb008d7459fc76cabb4f8b2224130/homeassistant/helpers/config_entry_oauth2_flow.py#L82

Reply:
meh...